sokutei.h

```diff
@@ -78,29 +93,46 @@ int sokutei_get_index_of_counter(const char *counter_name){
 
 
 void sokutei_set_number_of_iterations(const int iterations){
-
     if(SOKUTEI_MAX_MEASURED_ITERATIONS < iterations){
         //TODO call error
         return;
     }
-
     number_of_iterations = iterations;
 }
 
 
-int sokutei_add_counter(const char *counter_name, char type){
+inline int sokutei_is_matching_type(char type){
```

Comment:
Maybe, instead of `is_matching_type`, a more appropriate name could be `is_known_type`. Also, the `type` parameter can be set to `const`, right?

Even better, since this function is only used in conjunction with a negation (`!fn`), you may consider inverting it:

~~~~C
inline int sokutei_is_unknown_counter_type(const char type) {
  // Check if unknown.
}
~~~~